routes/v1.js

```diff
@@ -256,11 +256,13 @@ api.declare({
     "",
     "**Task specific routing-keys**, using the `task.routes` property you may",
     "define task specific routing-keys. If a task has a task specific ",
-    "routing-key: `<route>`, then the poster will be required to posses the",
-    "scope `queue:route:<route>`. And when the an AMQP message about the task",
```

Comment:
I don't really care... this seems nit to me... The added warning is nice though! :)